recipe/build.sh

```diff
@@ -68,6 +68,62 @@ if [[ "$blas_impl" == "accelerate" ]]; then
         -Wl,-reexport_library,$SRC_DIR/accelerate/liblapacke-netlib.${PKG_VERSION}.dylib
 
     cp libvecLibFort-ng.dylib $SRC_DIR/accelerate/
+
+elif [[ "$blas_impl" == "newaccelerate" ]]; then
+    # New Accelerate libraries have all symbols in BLAS, CBLAS, LAPACK with $NEWLAPACK
+    # name appended to all the symbol names. Therefore we create a library that dispatches
+    # eg: _dgemm -> _dgemm$NEWLAPACK with aliases.txt
+    #
+    # One exception to this is {c,z}dot{u,c}_ symbols which have different signatures.
+    # We use wrap_accelerate.c to fix those.
+    #
+    # For LAPACKE symbols, we use the LAPACKE wrappers from netlib which would call
+    # the LAPACK symbols from Accelerate.
+    #
+    # All of tese are exported from libblas_reexport.dylib
+
+    mkdir -p $SRC_DIR/accelerate
+    cp $NEW_ENV/lib/liblapacke.dylib $SRC_DIR/accelerate/liblapacke-netlib.${PKG_VERSION}.dylib
+    $INSTALL_NAME_TOOL -id "@rpath/liblapacke-netlib.${PKG_VERSION}.dylib" $SRC_DIR/accelerate/liblapacke-netlib.${PKG_VERSION}.dylib
+
+    veclib_loc=$SDKROOT/System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A
+    veclib_libblas="${veclib_loc}/libBLAS.tbd"
+    veclib_liblapack="${veclib_loc}/libLAPACK.tbd"
+    if [[ ! -f ${veclib_loc}/libLAPACK.tbd ]]; then
+        echo "could not find TBD file ${veclib_loc}/libLAPACK.tbd"
+	exit 1
+    fi
+
+    export LDFLAGS="${LDFLAGS/-Wl,-dead_strip_dylibs/}"
```

Comment:
All but the `libLAPACK.tbd` lines here are duplicated with the `accelerate` handling. Not sure it's easy to structure differently, and not very important, but I wanted to at least point it out.

Reply:
It's fine to have a bit of duplicating here.